dice.js

```diff
@@ -0,0 +1,2 @@
+let diceNumber = 1 + Math.floor(Math.random() * 6);
```

Comment:
Eres un titan bro pero cambia el let y el nombre de la variable <3. Arriba el rambla
```suggestion
const MIN_DICE = 1;
const  SIDE_DICE = 6;
const diceResult = MIN_DICE + Math.floor(Math.random() * SIDE_DICE);
```